index.js

```diff
@@ -1,40 +1,74 @@
 // ***************************************************************************
 // Iteration 1 - `for...of` loop
 // ***************************************************************************
-
+console.log("===== Iteration 1 ====== ");
 const getFirstNames = arr => {
   const userFirstNames = [];
   for (let user of arr) {
     // Your code goes here ...
+    userFirstNames.push(user.firstName);
   }
+  return userFirstNames;
 };
 
-getFirstNames(usersArray);
+console.log(getFirstNames(usersArray));
 // expected output:
 // [ 'Kirby', 'Tracie', 'Kendra', 'Kinney', 'Howard', 'Rachelle', 'Lizzie' ]
 
 // ***************************************************************************
 // Iteration 2 - `for...of` loop and ES6 string literals `${}`
 // ***************************************************************************
-
+console.log("===== Iteration 2 ====== ");
 const getFullNames = arr => {
+  const userFullNames = [];
   // Your code goes here ...
+  for (let user of arr) {
+    userFullNames.push(user.firstName + " " + user.lastName);
```

Comment:
Here you could have used string interpolation:
```js
userFullNames.push(`${user.firstName} ${user.lastName}`)
```

You could also destructure the firstName and lastName variable, but its still more than fine